vars/piperInitRunStageConfiguration.groovy

```diff
@@ -118,3 +120,17 @@ private static boolean checkExtensionExists(Script script, Map config, String st
     def globalInterceptorFile = "${config.globalExtensionsDirectory}${stageName}.groovy"
     return script.fileExists(projectInterceptorFile) || script.fileExists(globalInterceptorFile)
 }
+
+/**
+ Before syncing the piper-stage-config.yml file, there were differences in the display names of some stages.
+ This function duplicates the runStage and runStep values from the new stage name into the old one to ensure compatibility.
+ For example, the 'Build' is not used in Jenkins but is used by other orchestrators, whereas 'Central Build' is
+ used by Jenkins but unused by other orchestrators.
+ */
+private static void handleRenamedStages(Script script) {
+    script.commonPipelineEnvironment.configuration.runStage["Central Build"] = script.commonPipelineEnvironment.configuration.runStage["Build"]
+    script.commonPipelineEnvironment.configuration.runStep["Central Build"] = script.commonPipelineEnvironment.configuration.runStep["Build"]
+
+    script.commonPipelineEnvironment.configuration.runStage["Post Actions"] = script.commonPipelineEnvironment.configuration.runStage["Post"]
+    script.commonPipelineEnvironment.configuration.runStep["Post Actions"] = script.commonPipelineEnvironment.configuration.runStep["Post"]
```

Comment:
This requires the "Build" and "Post" stages to be defined (correctly), otherwise it will just overwrite the real stage config with an empty one.

Reply:
conditionals added